http/headers/Expect.json

```diff
@@ -7,22 +7,22 @@
           "spec_url": "https://www.rfc-editor.org/rfc/rfc9110#field.expect",
           "support": {
             "chrome": {
-              "version_added": null
```

Comment:
Where the data was originally null I'm less confident that setting it to "1" is correct, because nobody has ever claimed it is supported. Can you break this and any others like it into a separate PR for closer review?

Reply:
Reverted those changes!